### PR TITLE
Monitoring Doc

### PR DESCRIPTION
**Pull request by huntc**
Describes how to configure monitoring for ConductR.

### CODE REVIEW

File: src/main/play-doc/operation/Monitoring.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ConductR is a Typesafe RP based application and has been configured to support Typesafe Monitoring by enabling some settings within its `conf/application.ini`.
+
+### Enabling ConductR Monitoring for Takipi
+
+To enable monitoring for ConductR with [Takipi](https://www.takipi.com/) you'll need an account by [signing up with them](https://app.takipi.com/). Once done you can enable ConductR (you'll need to do this for each node where ConductR runs):
+
+``` bash
+echo \
+  -Dcinnamon.disabled=off
+  -J-agentlib:TakipiAgent
```
Comment (fsat) [inline src/main/play-doc/operation/Monitoring.md]: @huntc - won't you need `| \` for these lines?

Comment (huntc) [inline src/main/play-doc/operation/Monitoring.md]: oh

File: src/main/play-doc/operation/Monitoring.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  sudo tee -a /usr/share/conductr/conf/application.ini
+sudo /etc/init.d/conductr restart
+```
+
+conductr-haproxy can also be monitored:
+
+``` bash
+echo \
+  -Dcinnamon.disabled=off
+  -J-agentlib:TakipiAgent
```
Comment (fsat) [inline src/main/play-doc/operation/Monitoring.md]: @huntc - also `| \` for these lines?

File: src/main/play-doc/operation/Monitoring.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+This guide discusses ConductR specifics regarding Typesafe Monitoring. For a comprehensive description please visit [Typesafe Monitoring's documentation](http://resources.typesafe.com/monitoring/docs/home.html).
+
+## Monitoring ConductR
+
+ConductR is a Typesafe RP based application and has been configured to support Typesafe Monitoring by enabling some settings within its `conf/application.ini`.
+
+### Enabling ConductR Monitoring for Takipi
+
+To enable monitoring for ConductR with [Takipi](https://www.takipi.com/) you'll need an account by [signing up with them](https://app.takipi.com/). Once done you can enable ConductR (you'll need to do this for each node where ConductR runs):
```
Comment (fsat) [inline src/main/play-doc/operation/Monitoring.md:13]: @huntc - does the RP subscription not automatically enable Takipi?

Comment (huntc) [inline src/main/play-doc/operation/Monitoring.md:13]: Nope

File: src/main/play-doc/operation/Monitoring.md (hunk 2 above)
Comment (huntc) [inline src/main/play-doc/operation/Monitoring.md]: yep - good catch

**Comment by fsat** [comment]
LGTM - after `| \` is put in the commands

**Pull request merged**